antlr-kotlin-examples-jvm/build.gradle.kts

```diff
@@ -1,3 +1,16 @@
+val antlrVersion = "4.7.1"
+val antlrKotlinVersion = "0.0.5"
+
+buildscript {
+    // we have to re-declare this here :-(
+    val antlrKotlinVersion = "0.0.5"
```

Comment:
that is ok, as this is like it was a separate project, something a user of ANTLR Kotlin would use, so it makes sense they have to specify the version number of a dependency they are using